assets/src/components/form/select/model-selects/event.js

```diff
@@ -10,7 +10,7 @@ import { withBaseControl } from '../../../../higher-order-components';
  */
 import { __ } from '@eventespresso/i18n';
 import { Component } from '@wordpress/element';
-import PropTypes from 'prop-types';
+import { PropTypes } from 'prop-types';
```

Comment:
Why did this get changed?  It should be `import PropTypes from 'prop-types'`

Reply:
I changed this to match all of the other model select components as well as the base control which you yourself had done:

https://github.com/eventespresso/event-espresso-core/commit/2cec1974dbc2ef3b6e2b2b2f532bfa2e0d8902e2#diff-7c5379ae2262193feb579136c72d592a

I assumed the way you were performing that import was correct and so copied that.

Reply:
right, and any place you see it doing `import { Proptypes } from 'prop-types'` is also incorrect (regardless of who did it).  So we should make sure its corrected :)